src/app_engine/encryptor_test.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+
+import encryptor
+from encryptor import Encryptor
+from encryptor import Hasher
+from google.appengine.ext import testbed
+
+
+class EncryptorTest(unittest.TestCase):
+  """Tests the Encryptor class in the encryptor module."""
+
+  def setUp(self):
+    # First, create an instance of the Testbed class.
+    self.testbed = testbed.Testbed()
+
+    # Then activate the testbed, which prepares the service stubs for use.
+    self.testbed.activate()
+
+    # Override AES key for testing.
+    encryptor.AES_KEY = 'a' * 16
+    encryptor.HASH_SALT = 'a' * 16
+
+  def testEncryptAndDecrypt(self):
+    message = b'All your base are belong to us!'
```

Comment:
With the padding and unpadding code, it might make sense to test this with a couple of different length strings.